stories/Molecules/Text/Faq/Faq.scss

```diff
@@ -10,45 +10,52 @@
 
 .accordion__item {
   border-bottom: 1px solid $color-gray-500;
-  position: relative;
-  padding: $spacing-16 0 $spacing-24;
   cursor: pointer;
-  overflow: hidden;
   list-style: none;
+  overflow: hidden;
+  padding: $spacing-16 0 $spacing-24;
+  position: relative;
+
   &::before {
-    content: "";
+    @include transition(0.1s ease);
+
     background: url(#{$img-path-Icon}/Chevron-down.svg) no-repeat center center;
+    border-top-color: #f3f3f3;
```

Comment:
@harishkumar-srijan If we have the color variable for this then please use that

Reply:
We dont' have this variable and this color we never used again in the story.